Tools/machines/perlmutter-nersc/Containerfile

```diff
@@ -1,167 +1,291 @@
 # Build this container from its current directory:
-#   podman build --build-arg NJOBS=6 -t warpx-perlmutter-dev .
+#   podman build --build-arg NJOBS=6 -t warpx-perlmutter .
 # Adjust NJOBS to the number of processes to use for parallel compilation.
 #
-# Run from the via WarpX source directory
-#   podman run -v ${PWD}:/opt/warpx -it warpx-perlmutter-dev
-# then
-#   cd /opt/warpx
-# and compile.
+# WarpX executables are installed in /usr/bin/
+# WarpX Python bindings are installed in /opt/venv
+#
+# On Perlmutter, run WarpX like this:
+#   podman run --rm --gpu --mpi --nccl -it warpx-perlmutter warpx.2d inputs_2d
```

Comment:
Urgh, the `podman-hpc --mpi` plugin is broken on PM, fails in startup.

The moment I put `--mpi` or `--cuda-mpi` in, it starts to wildly connect to external registries and fails
<img width="924" height="446" alt="Screenshot from 2025-11-18 10-28-51" src="https://github.com/user-attachments/assets/d3c9766d-6c35-4f9e-a881-2d172a2bd6eb" />
https://docs.nersc.gov/development/containers/podman-hpc/overview/#using-podman-hpc-as-a-container-runtime